homeassistant/components/sensor/bmw_connected_drive.py

```diff
@@ -74,6 +83,16 @@ def unit_of_measurement(self) -> str:
         """Get the unit of measurement."""
         return self._unit_of_measurement
 
+    @property
+    def device_state_attributes(self):
+        """Return the state attributes of the binary sensor."""
+        vehicle_state = self._vehicle.state
+
+        return {
+            'last_update': vehicle_state.timestamp,
+            'car': self._vehicle.modelName
+        }
+
     def update(self) -> None:
         """Read new state data from the library."""
         _LOGGER.debug('Updating %s', self.entity_id)
```

Comment:
Replace this entity_id with name as in the other platforms.

Also please update the state update logic below as in the other platforms.

Reply:
Done